test/gtest/uct/test_atomic_key_reg_rdma_mem_type.cc

```diff
@@ -30,14 +30,41 @@ UCS_TEST_SKIP_COND_P(uct_atomic_key_reg_rdma_mem_type, fadd64,
                      !check_atomics(UCS_BIT(UCT_ATOMIC_OP_ADD), FOP64) ||
                      !check_rdma_memory())
 {
-    mapped_buffer recvbuf(sizeof(uint64_t), receiver(), 0UL,
-                          UCS_MEMORY_TYPE_RDMA);
+    mapped_buffer *recvbuf     = nullptr;
+    const size_t buffer_size   = sizeof(uint64_t);
```

Comment:
Minor: personally I don't see much sense in all of these constants, but up to you

Reply:
Removed "const"